src/wallet/feebumper.cpp

```diff
@@ -304,7 +361,8 @@ Result CreateRateBumpTransaction(CWallet* wallet, const uint256& txid, const CCo
     return Result::OK;
 }
 
-bool SignTransaction(CWallet* wallet, CMutableTransaction& mtx) {
```

Comment:
Nit: you're not touching this function, so don't fix the brackets please

Reply:
fixed :)